src/custom/utils/price.ts

```diff
@@ -413,9 +413,7 @@ export async function getGpUsdcPrice({ strategy, quoteParams }: Pick<QuoteParams
     quoteParams.validTo = MAX_VALID_TO_EPOCH
```

Comment:
Side note, unrelated to this fix: we should not send max epoch anymore.

Reply:
You can use:
```ts
{
  // ...
  validFor: 10 * 60, // 10 minutes
  // ...
}
```

https://github.com/cowprotocol/services/commit/29cee54830831726bec5e4120867292fd2dff4b2

Reply:
Oh right, there are relative times now! Awesome!

Reply:
I was not very aware of this either. By the way, i remember killing this MAX_VALID_TO...

Reply:
Likely your changes are parked on develop?

Reply:
mmm, I don't think so!

@W3stside is your branch updated?
https://github.com/cowprotocol/cowswap/pull/621

<img width="1504" alt="image" src="https://user-images.githubusercontent.com/2352112/174341267-666ad306-d619-432a-b6aa-0fad9ee89b87.png">

Reply:
Mmm, i've done my change in the hook that prepare all the params and do the call

https://github.com/cowprotocol/cowswap/blob/cca8a0840a9bb0ba10e582f28e9787db5bcd615c/src/custom/hooks/useUSDCPrice/index.ts#L106


Why there's this other override? https://github.com/cowprotocol/cowswap/blob/ec98a5b335e08320d6021b41fff8e3a0dd002a9e/src/custom/utils/price.ts#L413

Reply:
this hotfix is branched of 1.15.2

Reply:
you likely changed the MAX_EPOCH in another function, not this specific usd fetching fn

Reply:
removing this line